### PR TITLE
Allow specification of multiple hosts

### PR DESCRIPTION
**Pull request by joshuafleck**
The purpose of this change is to allow us to forgo the load balancer and allow bunny to manage the hosts directly. This allows us to pass a list of hosts to bunny, of which it will choose one to connect and will failover upon lost connection.

@sparrovv @calo81 @daniel-barlow @PaulMcAdam @jaynefox @cpoo22 Could you please sign off?

### CODE REVIEW

File: lib/rabbit_feed/configuration.rb [hunk 1]
```diff
@@ -2,13 +2,14 @@ module RabbitFeed
   class Configuration
     include ActiveModel::Validations
 
-    attr_reader :host, :port, :user, :password, :application, :environment, :exchange, :pool_size, :pool_timeout, :heartbeat, :connect_timeout, :network_recovery_interval, :auto_delete_queue, :auto_delete_exchange
+    attr_reader :host, :hosts, :port, :user, :password, :application, :environment, :exchange, :pool_size, :pool_timeout, :heartbeat, :connect_timeout, :network_recovery_interval, :auto_delete_queue, :auto_delete_exchange
```
Comment (calo81) [inline lib/rabbit_feed/configuration.rb]: is singular `host` still relevant?

**Comment by sparrovv** [comment]
I think it makes sense to update README with that new setting.

File: lib/rabbit_feed/configuration.rb [hunk 2]
```diff
@@ -42,6 +43,7 @@ def connection_options
         options[:heartbeat] = heartbeat if heartbeat
         options[:connect_timeout] = connect_timeout if connect_timeout
         options[:host] = host if host
+        options[:hosts] = hosts if hosts
```
Comment (calo81) [inline lib/rabbit_feed/configuration.rb]: sorry,  ask cause I don't know how this works... is this property being picked up automatically by `Bunny`?.. I mean where is this value actually used?

File: lib/rabbit_feed/configuration.rb (hunk 1 above)
Comment (joshuafleck) [inline lib/rabbit_feed/configuration.rb]: Yes, for connecting when there is not a cluster

**Comment by joshuafleck** [comment]
@sparrovv 

> I think it makes sense to update README with that new setting.

Added in 516e6fa, thanks!

File: lib/rabbit_feed/configuration.rb (hunk 2 above)
Comment (joshuafleck) [inline lib/rabbit_feed/configuration.rb]: @calo81 

>  is this property being picked up automatically by Bunny?

Yes

> I mean where is this value actually used?

`connection_options` are what is passed to Bunny, see: https://github.com/simplybusiness/rabbit_feed/blob/master/lib/rabbit_feed/connection_concern.rb#L53

Bunny is what is actually uses the configs. RabbitFeed allows you to specify configs in yaml format and passes them to Bunny for you.

**Comment by calo81** [comment]
ok.. thanks for letting me know. It all looks good to me... What about the codeclimate in red?. anything to be aware of or is it being like that always?

**Comment by joshuafleck** [comment]
> What about the codeclimate in red?. anything to be aware of or is it being like that always?

Codeclimate appears to be having connectivity issues with github, as @gzzsound and I encountered this on another project today.

**Comment by sparrovv** [comment]
Signed-off-by: @sparrovv

**Pull request merged**